code/tests/core/test_identityModule.brs

```diff
@@ -20,203 +20,94 @@ end sub
 ' @Test
 sub TC_adb_IdentityModule_init()
     configurationModule = _adb_ConfigurationModule()
+    identityState = _adb_IdentityState()
     consentState = _adb_ConsentState(configurationModule)
-    identityModule = _adb_IdentityModule(configurationModule, consentState)
+    edgeModule = _adb_EdgeModule(configurationModule, identityState, consentState)
+    identityModule = _adb_IdentityModule(identityState, edgeModule)
```

Comment:
Can you add test cases for IdentityModule.processResponseEvent? I'm not seeing them here.

Reply:
@kevinlind updated!